mxcube3/routes/beamline.py

```diff
@@ -1,149 +1,147 @@
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-
 import json
 import sys
 import logging
 
-from flask import Response, jsonify, request, make_response
+from flask import Blueprint, Response, jsonify, request, make_response
 
-from mxcube3 import server
-from mxcube3 import blcontrol
 from mxcube3.core import beamlineutils
 
+def init_routes(mxcube):
```

Comment:
attempt to isolate the routes and make them less dependent on a global import, please let me know what you think about this ?

Reply:
are you suggesting to do the same for all? I mean completing following method?
``` 
    def register_routes(mxcube):
        from mxcube3.routes.beamline import init_routes as init_beamline_routes
        Server.FLASK.register_blueprint(init_beamline_routes(mxcube))
```

Reply:
Yes, I just started with one route as we might need to discuss before we arrive to something final

Reply:
its a good start